runtime/web_storage_api.md

```diff
@@ -3,7 +3,8 @@
 Deno 1.10 introduced the
 [Web Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API)
 which provides an API for storing string keys and values. Persisting data works
-similar to a browser. The global `sessionStorage` object only persists data for
+similar to a browser, except its storage limit (at most 10MB). The global
```

Comment:
that "except" is inaccurate: chrome also has a 10MB limit. it would be just more appropraite to say "similar to a browser, and has a 10MB storage limit."

Reply:
Sounds good